v2/definitions/VideoJobResponse.js

```diff
@@ -0,0 +1,217 @@
+module.exports = {
+  type: 'object',
+  description: 'The response to a video request',
+  required: ['id', 'createdAt', 'status', 'video', 'fps', 'jobResults', 'imageResults'],
+  properties: {
+    id: {
+      type: 'string',
+    },
+    createdAt: {
+      type: 'integer',
+      format: 'int64',
+      description: 'UNIX timestamp for when the job was created',
+    },
+    status: {
+      $ref: '#/definitions/JobStatus',
+    },
+    video: {
+      type: 'string',
+      description: 'URL of the video',
+      maxLength: 512,
+    },
+    fps: {
+      type: 'integer',
+      description: 'Number of frames classified, per second.',
```

Comment:
Does `maximum` need to be specified here? Same as the request

Reply:
Yes it does.

Reply:
I specified the maximum in `VideoJobRequest.js` but not here.. I'll add it.

Reply:
Doesn't look like responses usually include limits like `maxLength` or `maximum`. Maybe remove it from video attribute?

Reply:
Okay. Will do.